widget/embedded/src/store/slices/settings.ts

```diff
@@ -131,6 +139,16 @@ export const createSettingsSlice: StateCreator<
     set((state) => ({
       infiniteApprove: !state.infiniteApprove,
     })),
+  setCustomTokens: (token) =>
```

Comment:
A token is being added to the list of custom tokens so maybe `setCustomTokens` name is not suitable for this.

Reply:
I change to setCustomToken

Reply:
Maybe 'add' or 'insert' is better

Reply:
Thank you , fixed

Reply:
I think it is still 'setCustomToken'